### DISCUSSION TITLE
ERR on runnimg npm install

### DISCUSSION THREAD

**Discussion by Tanay-1009** [LalanaChami/Pharmacy-Mangment-System#250] [Q&A]
Versions
VSCodeUserSetup-x64-1.83.0
node-v18.18.0-x64

![Screenshot (5)](https://github.com/LalanaChami/Pharmacy-Mangment-System/assets/102869781/fe20764c-0201-4107-9329-21753dce720b)

**Comment by Yuvindu** [comment]
You need to tick desktop development with C++ option when downloading Visual Studio. Then it should work.
